src/sentry/api/endpoints/organization_details.py

```diff
@@ -271,6 +273,13 @@ def validate(self, attrs):
                 raise serializers.ValidationError(
                     {"avatarType": "Cannot set avatarType to upload without avatar"}
                 )
+        # both provider and providerConfig required to configure provider
+        if ("provider" in attrs and "providerConfig" not in attrs) or (
+            "providerConfig" in attrs and "provider" not in attrs
+        ):
```

Comment:
```suggestion
        if !("provider" in attrs and "providerConfig" in attrs):
```

Reply:
`provider` and `providerConfig` are optional so it shouldn't error if both of them aren't passed in

Need something like XOR opertaions for booleans so I could do something like this to make it less verbose but idk if that's very clean 
https://stackoverflow.com/questions/432842/how-do-you-get-the-logical-xor-of-two-variables-in-python

Reply:
Wait jk I can do (boolean ^ boolean) for XOR which is pretty clean